Ecommerce/index.html

```diff
@@ -1,13 +1,32 @@
-<!DOCTYPE html>
-<html>
-  <head>
-    <title>E-commerce</title>
-    <meta charset="UTF-8" />
-    <link rel="stylesheet" href="index.css" />
```

Comment:
The image source 'shp.png' is relative. It may cause issues if the file is not located in the same directory as the HTML file.